packages/utilities/src/CliUtils.ts

```diff
@@ -210,8 +221,17 @@ export class CliUtils {
         let args: ICommandArguments["args"] = {};
         options.forEach((opt) => {
             let envValue: any = CliUtils.getEnvValForOption(envPrefix, opt.name);
-            if (envValue != null) {
 
+            if (envValue == null) {
+                // Use aliases for backwards compatibility
+                // Search for first alias available in the process.env
+                if ("aliases" in opt) {
+                    const oldOption = opt.aliases.find(o => CliUtils.getEnvValForOption(envPrefix, o) != null);
+                    if (oldOption) envValue = CliUtils.getEnvValForOption(envPrefix, oldOption);
+                }
+            }
```

Comment:
I would vote not to check aliases for environment variables, since it could lead to conflicts as you mentioned, and I believe the enhancement request we received was specifically for v1 profiles. Curious what others' thoughts are? 🙂

Reply:
Will proceed with not lookin at ENV vars 😋